### PR TITLE
Add 0.7 and 1.0 in travis and appveyor

### PR DESCRIPTION
**Pull request by abelsiqueira**

### CODE REVIEW

**Comment by dpo** [comment]
LDL depends on AMD.

**Comment by abelsiqueira** [comment]
Yes, only saw that after. I'm looking into that error.

**Comment by abelsiqueira** [comment]
Needs https://github.com/JuliaLang/METADATA.jl/pull/16986

**Comment by dpo** [comment]
Shoud use AMD 2.0.1 (merged into METADATA).

File: appveyor.yml [hunk 1]
```diff
@@ -4,9 +4,20 @@ environment:
   - JULIA_URL: "https://julialang-s3.julialang.org/bin/winnt/x64/0.5/julia-0.5-latest-win64.exe"
   - JULIA_URL: "https://julialang-s3.julialang.org/bin/winnt/x86/0.6/julia-0.6-latest-win32.exe"
```
Comment (dpo) [inline appveyor.yml]: Need to remove julia 0.5.

File: .travis.yml [hunk 2]
```diff
@@ -5,10 +5,16 @@ os:
...
-  - 0.5
   - 0.6
+  - 0.7
+  - 1.0
   - nightly
 
+matrix:
+  allow_failures:
+    - 1.0
+    - nightly
```
Comment (dpo) [inline .travis.yml]: For some reason, these don't appear as allowed failures on Travis.

**Comment by dpo** [comment]
I pulled your changes from here into #5. Closing this. Thank you!

**Pull request closed without merging**